src/python/zquantum/optimizers/daemon_optimizer/optimize/_api.py

```diff
@@ -38,65 +36,19 @@ def optimize_variational_circuit_with_proxy(initial_params, optimizer, client, *
 
     opt_results = scipy.optimize.OptimizeResult({'value': None, 'status': None, 'success': False, 'nfev': 0,
         'nit': 0, 'history':[{'optimization-evaluation-ids': []}]})
```

Comment:
I believe lines 32 through 38 are all unnecessary given the changes.

Reply:
Addressed this in [`203825`](https://github.com/zapatacomputing/z-quantum-optimizers/commit/203825750c6cc6261fa29aec3331333e475238d3)